mlflow/server/js/src/model-registry/components/ModelVersionTable.tsx

```diff
@@ -57,7 +57,7 @@ type ModelVersionTableProps = {
   modelEntity?: ModelEntity;
   onMetadataUpdated: () => void;
   usingNextModelsUI: boolean;
-  aliases?: ModelAliasMap
+  aliases?: ModelAliasMap;
```

Comment:
prettier updated this file

Reply:
that's weird.

Reply:
it looks like we don't run `yarn prettier` in `js.yml`.